cdh5/etc/oozie/conf/oozie-env.sh

```diff
@@ -31,9 +31,9 @@ export CATALINA_BASE=/var/lib/oozie/tomcat-deployment
 # Java System properties for Oozie should be specified in this variable
 #
 export OOZIE_HTTPS_PORT=11443
-export OOZIE_HTTPS_KEYSTORE_PASS=password
-export CATALINA_OPTS="$CATALINA_OPTS -Doozie.https.port=${OOZIE_HTTPS_PORT}"
-export CATALINA_OPTS="$CATALINA_OPTS -Doozie.https.keystore.pass=${OOZIE_HTTPS_KEYSTORE_PASS}"
+export OOZIE_HTTPS_KEYSTORE_FILE=/etc/oozie/conf/oozie.keystore
+export OOZIE_HTTPS_KEYSTORE_PASS=hadoop
```

Comment:
Yeah, kind of.  I had already set the pass to be 'hadoop' for hdfs/yarn/mapred, and I didn't want to go back and change it to something else. I'm using the same keystore for both oozie / hadoop, it's just copied to a different dir for oozie.